dali/core/fast_div_test.cu

```diff
@@ -130,7 +130,7 @@ DEVICE_TEST(FastDiv, DISABLED_U64_GPU_Slow, dim3(1<<10, 1<<10), (1<<10)) {
     auto ref = value / divisor;
     auto result = value / fast;
     if (result != ref) {
-      printf("%lld / %lld   got %lld expected %lld\n", value, divisor, result, ref);
+      printf("%lu / %lu   got %lu expected %lu\n", value, divisor, result, ref);
```

Comment:
Why not `ll`? There are 64-bit platforms with 32-bit long.... (Windows anyone?). I know we don't officially support it but at least we could refrain from leaving traps.

Reply:
Cause it's unsigned, if anything than `llu`.

Reply:
That's my point. long long, not long.